aws-android-sdk-core/src/main/java/com/amazonaws/regions/RegionMetadataParser.java

```diff
@@ -109,9 +109,9 @@ private static List<Region> internalParse(
         Document document;
         try {
 
-            DocumentBuilderFactory factory =
-                    DocumentBuilderFactory.newInstance();
-
+            DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
```

Comment:
why not use `getDocumentBuilderFactory`?

Reply:
that function does not exist in this file. I've only added it in the other file